Data/Set.hs

```diff
@@ -13,14 +13,48 @@
 -- Maintainer  :  libraries@haskell.org
 -- Portability :  portable
 --
--- An efficient implementation of sets.
+--
+-- = Sets
+--
+-- The @'Set' e@ type represents a set of /unique/, /ordered/ elements of type
+-- @e@.
+--
+-- For a walkthrough of the most commonly used functions see the
+-- <https://haskell-containers.readthedocs.io/en/latest/set.html sets introduction>.
+--
+-- Note that the implementation is /left-biased/ -- the elements of a
+-- first argument are always preferred to the second, for example in
+-- 'union' or 'insert'.  Of course, left-biasing can only be observed
+-- when equality is an equivalence relation instead of structural
+-- equality.
```

Comment:
See my note on `Map`. Also, rather than "left-biasing", how about "this bias"?

Reply:
Done.